src/logic/safe/transactions/gasNew.ts

```diff
@@ -25,33 +28,40 @@ const estimateDataGasCosts = (data) => {
   return data.match(/.{2}/g).reduce(reducer, 0)
 }
 
-export const estimateTxGasCosts = async (safeAddress, to, data, tx?: any, preApprovingOwner?: any) => {
+export const estimateTxGasCosts = async (
+  safeAddress: string,
+  to: string,
+  data: string,
+  tx?: Transaction,
+  preApprovingOwner?: string,
+): Promise<number> => {
   try {
     const web3 = getWeb3()
     const from = await getAccountFrom(web3)
-    const safeInstance: any = new web3.eth.Contract(GnosisSafeSol.abi as any, safeAddress)
+    const safeInstance = (new web3.eth.Contract(
+      (GnosisSafeSol.abi as unknown) as AbiItem,
```

Comment:
perhaps `unknown[]`

Reply:
can you move the Contract arguments to a constant?

Reply:
what do you mean by contract arguments? also I didn't understand `unknown[]` part

Reply:
`abi`, shouldn't it be an array?

Reply:
I mean moving the arguments of web3.eth.Contract to a constant.

Reply:
Sorry, could you please provide an example?

So far I can only think of extract abi, something like this:
```
const GnosisSafeAbi = (GnosisSafeSol.abi as unknown) as AbiItem
```
is this what you mean?

Reply:
yes, that!

Reply:
Ah, didn't see this. It should have been `AbiItem[]` instead. I gave my feedback in `safeContracts.ts`

Reply:
I think it doesn't make much sense as we would still have to import GnosisSafeSol in other files, in `safeContracts.ts` we also use `.bytecode` and `.network` properties